modules/util/buildhelpers.py

```diff
@@ -976,4 +976,67 @@ def move_docs(module_docs_path):
+    "Logon Session":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/logon_session.yml",
+    "Module":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/module.yml",
+    "Named Pipe":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/named_pipe.yml",
+    "Network Share":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/network_share.yml",
+    "Network Traffic":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/network_traffic.yml",
+    "Pod":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/pod.yml",
+    "Process":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/process.yml",
+    "Scheduled Job":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/scheduled_job.yml",
+    "Script":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/script.yml",
+    "Sensor Health":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/sensor_health.yml",
+    "Service":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/service.yml",
+    "Snapshot":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/snapshot.yml",
+    "User Account":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/user_account.yml",
+    "Volume":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/volume.yml",
+    "Web Credential":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/web_credential.yml",
+    "Windows Registry":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/windows_registry.yml",
+    "WMI":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/wmi_object.yml"
+}
+
+def data_sources_link(data_sources):
+    """ Data source mapping for April release """
+
+    link = "<a target='_blank' href='{}'>{}</a>"
+
+    def replace_link(data_sources):
+        
+        updated_data_sources = []
+
+        for data_source in data_sources:
+            data_source_str = data_source
+            if ":" in data_source:
+                data_source_name = data_source.split(":")[0]
+                data_source_component = data_source.split(":")[1]
+
+                if new_data_source_mapping.get(data_source_name):
+                    data_source_str = f"{link.format(new_data_source_mapping.get(data_source_name), data_source_name)}: {data_source_component}"         
+            else:
+                if new_data_source_mapping.get(data_source):
+                    data_source_str = f"{link.format(new_data_source_mapping.get(data_source), data_source)}" 
+
+            updated_data_sources.append(data_source_str)
+        
+        return updated_data_sources
+    
+    return ", ".join(replace_link(data_sources))
```

Comment:
Since data sources are only relevant to techniques, shouldn't this be stored under `modules/techniques/techniques.py`?